CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ Mapbox welcomes participation and contributions from everyone.
 ### Changelog
 [Changes between v2.9.0-beta.1 and v2.9.0-beta.2](https://github.com/mapbox/mapbox-navigation-android/compare/v2.9.0-beta.1...v2.9.0-beta.2)
 
+#### Known issues :bangbang:
+- Telemetry stops working due to internal deadlock. 
```

Comment:
@RingerJK , @LukasPaczos , do you think we need to mention that?

Reply:
seems it's not the issue, `CORESDK-1401`
`False alarm (misusage of the telemetry object instantiation).` cc @brunoabinader